k8s/aat/common/unspec/identity.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: unspec
+  namespace: unspec
+spec:
+  type: 0
+  ResourceID: /subscriptions/96c274ce-846d-4e48-89a7-d528432298a7/resourceGroups/managed-identities-aat-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/unspec-aat-mi
```

Comment:
Did u create this ? This isn't just about renaming in here, the resource should exist.

Reply:
@adusumillipraveen removed for now